lab_1_keywords_tfidf/main.py

```diff
@@ -156,4 +249,18 @@ def extract_significant_words(chi_values: dict[str, float], alpha: float) -> Opt
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    criterion = {0.05: 3.842, 0.01: 6.635, 0.001: 10.828}
+    if not (
+            isinstance(chi_values, dict)
+            and isinstance(alpha, float)
+            and type_of_elements(chi_values, tuple, str, float)
+            and chi_values
+            and alpha in criterion.keys()
+    ):
```

Comment:
`alpha in criterion.keys()` includes  `isinstance(alpha, float)`, so please remove a redundant check